_episodes/02-common-mistakes.md

```diff
@@ -199,11 +266,6 @@ that are excessively long. Including the units in the field names avoids confusi
 	<td> CellType </td>
 	<td> Cell Type </td>
 </tr>
-<tr>
-	<td> Observation_01 </td>
-	<td> first_observation</td>
-	<td> 1st Obs</td>
-</tr>
```

Comment:
What is the reason for this row being removed in another table?

Reply:
@tobyhodges if it was removed, it was a mistake.  I copied the other table to start a framework for the new table. I deleted an extra row when done. Hopefully I didn't actually delete that line from the other table, but it's possible. I'll certainly look in to this while considering the comments from the other maintainers.

Reply:
I've added back the lines to the other table. Hopefully this takes care of the first issue (making the table into HTML).